SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2493,6 +2498,13 @@ void ActorSystem::OnEntityCreated(const Worker_CreateEntityResponseOp& Op, FSpat
 		check(ClientSystemEntityId != SpatialConstants::INVALID_ENTITY_ID);
 		CommandsHandler.ClaimPartition(NetDriver->Connection->GetCoordinator(), ClientSystemEntityId, Op.entity_id);
 	}
+
+	if (EntitiesMapToRefreshDormancy.Contains(EntityId))
```

Comment:
Additional check that this was an actual successful creation, ie. check status_code == WORKER_STATUS_CODE_SUCCESS